packages/react-components/react-charts-preview/library/package.json

```diff
@@ -37,6 +37,7 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@fluentui/react-button": "^9.3.89",
```

Comment:
ensure that the build is passing before merging your changes

Reply:
yes build is passing, checked